src/clocksi_vnode.erl

```diff
@@ -150,15 +150,7 @@ get_active_txns_internal(TableName) ->
                     [] ->
                         [];
                     [{Key1, List1} | Rest1] ->
-                        lists:foldl(fun({_Key, List}, Acc) ->
-                            case List of
-                                [] ->
-                                    Acc;
-                                _ ->
-                                    List ++ Acc
-                            end
-                        end,
-                            [], [{Key1, List1} | Rest1])
+                        lists:flatmap(fun({_, _List}) -> _List end, [{Key1, List1} | Rest1])
```

Comment:
Why `_List`? Couldn't it be simply `List`?

Reply:
Just committed a fix: https://github.com/AntidoteDB/antidote/pull/414/commits/cb6ce9fc54e26929a16250b8fc33a487e5905154